### DISCUSSION TITLE
Releasing 2.7

### DISCUSSION THREAD

**Discussion by phase1geo** [phase1geo/tke#3] [General]
@aplsimple Alex, I want to spend some time working on TKE again and was wondering if the current state of the master branch is considered healthy enough to create a 2.7 release out of.  I plan to spend my time on the etke branch (which is not ready for a stable release).

**Comment by aplsimple** [comment]
Hi Trevor,

Glad to receive your e-mail.

It's okay with the current state of TKE on Github, at least I'd tried an
empty commit some hours ago and it went OK.

While TKE being suspended, I got an idea of my own editor on which I am
working just now. It's small and modest (esp. compared with TKE and
others), but it would have just those features I need mostly and have not
those I need not. Remember your mood at the starting TKE project, it's
nearly the same.

In no way this means I've forgotten TKE, just it seemed to me you had. Now
that it isn't the case, we can try and continue TKE. More exactly, it's you
who will continue while I would mess along with my little additions.

One thing is especially disturbing me, namely my plugins in Windows
stand-alone TKE.exe. I cannot test them with the TKE executable because I
haven't any notions about how you build it. Might you be so kind to hint me
on the tools / workflows / commands etc. you use?

When building my own Tcl/Tk executables, I don't use Freewrap as you
probably do.

Regards,
Alex
